tests/parsing_attributes.cpp

```diff
@@ -327,12 +327,25 @@ TEST_CASE("Parsing Attribute: Attributes (success)", "[parsing]") {
 
         // yes optional vector directory value documentation
         std::optional<std::vector<std::filesystem::path>> yesOptionalVectorDirectoryValue [[codegen::directory(true)]];
+        
```

Comment:
I'd suggest to also to copy some of these tests into an execution file to actually check the codegeneration of it as well.  That way you can check the resulting struct against Dictionarys as well